Gems/Atom/RPI/Code/Include/Atom/RPI.Public/DynamicDraw/DynamicBuffer.h

```diff
@@ -47,8 +47,8 @@ namespace AZ
             //! Get the buffer's size
             uint32_t GetSize();
 
-            //! Get the buffer's address. User can write data to the address. 
-            void* GetBufferAddress();
+            //! Get the buffer's address. User can write data to the address.
+            AZStd::unordered_map<int, void*> GetBufferAddress();
```

Comment:
Return const-ref instead of value? Also, since it returns multiple addresses, should the name be `GetBufferAddresses`?